Gruntfile.js

```diff
@@ -161,7 +161,23 @@ module.exports = function(grunt) {
           money: standing.money
         });
       })
-    })
+    });
+
+    players = _.mapObject(players, function(player) {
+      return {
+        id: player.id,
+        player: player.name,
+        tournaments: _.sortBy(player.tournaments, function(tournament) {
```

Comment:
@czarandy Pretty nasty here :/ Maybe we should save our tournament dates as standard unix time?